debian/autobake-deb.sh

```diff
@@ -173,13 +174,26 @@ echo "Creating package version ${VERSION} ... "
 # during the entire build process (safe because a build can always be restarted)
 if which eatmydata > /dev/null
 then
-  BUILDPACKAGE_PREPEND=eatmydata
+  BUILDPACKAGE_PREPEND="yes"
```

Comment:
This change and line 196 seems a bit complex. Why not continue using a simple variable that is either empty or has `eatmydata` in it?

Reply:
This is actually how shellcheck wants it. I think I'll rework it as it's not cleaner or better than it was before but I think I can make it shellcheck proof without that change as I don't like it by myself (Did it any way :-1: )